app/views/hub/organizations/index.html.erb

```diff
@@ -11,59 +11,25 @@
 
     <div class="grid">
       <div class="grid__item width-one-half">
-        <% if @coalitions.count > 0 %>
-          <p><%= t(".organizations_by_coalition") %></p>
-
-          <% @coalitions.each do |coalition| %>
-            <div class="grid-flex space-between center-aligned spacing-above-10">
-              <h2 class="spacing-below-0"><%= coalition.name %></h2>
-              <% unless coalition.organizations.count.zero? %>
-                <div class="help-text">Capacity</div>
-              <% end %>
-            </div>
+        <% @organizations_by_coalition.each do |coalition, organizations| %>
+          <div class="grid-flex space-between center-aligned spacing-above-10">
+            <h2 class="spacing-below-0"><%= coalition&.name || (t(".independent_organizations", count: @organizations_by_coalition.length)) %></h2>
```

Comment:
I think in this case, `<%= coalition.nil?  ? t(".independent_organizations", count: @organizations_by_coalition.length) : coalition.name %>` would be a nice way to use an explicit nil check.

What you have is fine, too.